### PR TITLE
Added nested folders creation

### PR DESCRIPTION
**Pull request by m-abramovich**

### CODE REVIEW

**Comment by pgbidkar** [comment]
Hi @m-abramovich , Please can you provide -S command line argument to skip Cert Verification?

**Comment by m-abramovich** [comment]
> Hi @m-abramovich , Please can you provide -S command line argument to skip Cert Verification?

Hey @pgbidkar. 
I do. But maybe it will take a time - I am very busy and have no VMware environment so far.
We can do it that way: I will add the flag within the few hours and you will test. Sounds reasonable?

**Comment by pgbidkar** [comment]
> > Hi @m-abramovich , Please can you provide -S command line argument to skip Cert Verification?
> 
> Hey @pgbidkar.
> I do. But maybe it will take a time - I am very busy and have no VMware environment so far.
> We can do it that way: I will add the flag within the few hours and you will test. Sounds reasonable?

@m-abramovich Sounds Good

**Comment by m-abramovich** [comment]
> Hi @m-abramovich , Please can you provide -S command line argument to skip Cert Verification?

pgbidkar, please check.
Is it working now?
It is either "-C" or "--no-cert"

**Comment by pgbidkar** [comment]
> > Hi @m-abramovich , Please can you provide -S command line argument to skip Cert Verification?
> 
> pgbidkar, please check.
> Is it working now?
> It is either "-C" or "--no-cert"

@m-abramovich  Thanks for quick fix. 
Some travis pipeline checks failed. Please fix them
Instead of -C or --no-cert, please follow the standard followed by all Samples. All samples use -S for skip cert verification.
Thanks

**Comment by m-abramovich** [comment]
@pgbidkar 
Here you go :-)

**Comment by pgbidkar** [comment]
> @pgbidkar
> Here you go :-)

Hey @m-abramovich  Sample is failing with error 'ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:661)'

File: samples/create_folder_in_datacenter.py [hunk 1]
```diff
@@ -65,10 +91,20 @@ def main():
         password = getpass.getpass(prompt='Enter password for host %s and '
                                    'user %s: ' % (args.host, args.user))
 
+    if args.nocert:
+        print("WARNING: SSL certificate check is skipped!")
+        s_context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
```
Comment (pgbidkar) [inline samples/create_folder_in_datacenter.py]: I think we can use ssl._create_unverified_context() or connect.SmartConnectNoSSL() method if Cert validation is skipped

Comment (m-abramovich) [reply samples/create_folder_in_datacenter.py]: @pgbidkar Sorry for long response awaiting.
I did that change. Please check if it works (as I mentioned above, I don't have VMware env).

**Comment by m-abramovich** [comment]
PR was cancelled as it was not checked for 5 years. Code was significantly changed ever since and led to conflicts.

**Pull request closed without merging**